virttest/utils_test/__init__.py

```diff
@@ -989,6 +992,7 @@ def env_check(self):
                 logging.error("Failed to restart libvirtd inside guest")
                 return False
             pip_pack = ['setuptools', 'netifaces', 'aexpect', 'netaddr']
+            pip_pack.insert(0, self.pip_bin)
             cmd = ""
             for each in pip_pack:
                 cmd += "%s install %s --upgrade;" % (self.pip_bin, each)
```

Comment:
not related to this PR but it'd make sense to use `&&` instead of `;` here as we are assuming PASS/FAIL based on the last command only...

Reply:
@ldoktor 
Lukas, maybe it better to run all arguments in one row without a cycle?

Reply:
IIRC at least some pip versions had problems installing some deps together, but with `pip upgrade pip` it should probably work. Do you want to include this in the current PR or are you going to send a followup PR?